src/js/pages/Setup.js

```diff
@@ -464,12 +513,49 @@ class Setup extends mixin(StoreMixin) {
           validationErrorText: this.getErrors('ip_detect_script'),
           value: this.state.formData.ip_detect_script
         }
-      ]
+      ],
+      {
+        fieldType: 'checkboxMultiple',
+        name: 'nested_cluster_options',
+        formRowType: 'inline',
+        value: [
+          {
+            name: 'telemetry_enabled',
+            label: this.getCheckboxLabel('Send Anonymous Telemetry', (<span>
```

Comment:
Why can't `getCheckboxLabel` just return the tooltip content too? Aren't they exactly the same?